src/theme/mantineTheme.ts

```diff
@@ -97,6 +98,11 @@ export const theme = createTheme({
         maw: 300,
         multiline: true
       }
+    }),
+    Divider: Divider.extend({
+      defaultProps: {
+        'aria-hidden': true
```

Comment:
personally i can't make the screen reader to actually read out the divider, but I guess it doesnt hurt to have this anyway